sample_factory/algo/learning/batcher.py

```diff
@@ -145,6 +145,8 @@ def stop(self):
         ...
 
     def init(self):
+        init_file_logger(experiment_dir(self.cfg))
```

Comment:
There are way too many calls to init_file_logger.
There should be only one per process. See the process init functions in parallel_runner.

As far as I know, we only need these:
1) learner worker process
2) rollout worker process
3) inference process
4) and of course runner itself (main process)

All of the other calls are redundant and should be removed. This includes calls in init() functions